modules/rosa-common-commands.adoc

```diff
@@ -5,7 +5,7 @@
 
 [id="rosa-common-commands_{context}"]
 = Common commands and arguments
-
+:_content-type: REFERENCE
```

Comment:
@mletalie Thanks for adding these. The content type attribute should sit before the anchor ID. [1]

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata